### ISSUE TITLE
Use Linked Data Platform collection (and related) patterns

### ISSUE THREAD

**Issue by spyysalo** [restful-open-annotation/spec#5]
As of about two weeks ago there is now a W3C recommendation covering things like how to implement the collection pattern for linked data: (from http://www.w3.org/)

> ### Linked Data Platform 1.0 is a W3C Recommendation
> 
> 26 February 2015 | Archive
> 
> [The Linked Data Platform (LDP) Working Group](http://www.w3.org/2012/ldp/) has published a W3C Recommendation of [Linked Data Platform 1.0](http://www.w3.org/TR/2015/REC-ldp-20150226/).

**Comment by spyysalo** [comment]
OK, this sucks: http://www.w3.org/TR/2015/REC-ldp-20150226/#h-ldprs-get-turtle

> 4.3.2.1 LDP servers MUST respond with a Turtle representation of the requested LDP-RS when the request includes an Accept header specifying text/turtle, unless HTTP content negotiation requires a different outcome [turtle].

It's hard enough to get people to add `@context` to their JSON, leading off with "this is a turtle-based API, but ..." will have them zoning out before you can say "content negotiation".

IMHO this is a dealbreaker for trying to follow the LDP spec in REST-OA core, but we could get conformance using a simple piece of middleware.

**Comment by akuckartz** [comment]
Yes, the LDP Working Group perhaps should better have selected JSON-LD as the primary RDF serialization. Wilful violations of the requirement to support Turtle can be considered.
